dev.ksylvestre.MightyGrepDemo.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>dev.ksylvestre.MightyGrepDemo</id>
```

Comment:
`ksylvestre.dev` doesn't exist. the application id doesn't meet the requirements.